include/miopen/miopen.h

```diff
@@ -447,7 +448,18 @@ typedef enum
     miopenLRNWithinChannel = 0, /*!< Channel independent */
     miopenLRNCrossChannel  = 1, /*!< Cross Channel */
 } miopenLRNMode_t;
-
+#ifdef MIOPEN_BETA_API
```

Comment:
API changes require serious review and explicit approval from the key developers.

/cc @junliume @JehandadKhan @pfultz2